### ISSUE TITLE
Create accessibility page

### ISSUE THREAD

**Issue by xNS5** [xNS5/rentalreviews#67]
I want to create a page that displays information about accessibility on the site. Nothing super complicated, should be as simple as the FAQ/Privacy policy. Just need to throw together some words and create a JSON file in the data repo.

Inspiration:

* https://www.cnn.com/accessibility
* https://www.facebook.com/help/273947702950567
* https://www.section508.gov/manage/laws-and-policies/website-accessibility-statement/
*
